lighthouse-core/audits/dobetterweb/dom-size.js

```diff
@@ -64,11 +64,11 @@ class DOMSize extends Audit {
    */
   static get defaultOptions() {
     return {
-      // 25th and 50th percentiles HTTPArchive -> 50 and 75
+      // 25th and 50th percentiles HTTPArchive -> median and derived p10.
```

Comment:
```suggestion
      // 75th and 50th percentiles HTTPArchive -> median and derived p10.
```

? or they need to be flipped order?

Reply:
> ? or they need to be flipped order?

I might just delete the ones that are super ambiguous like this one :)